sql/core/src/test/resources/sql-functions/sql-expression-schema.md

```diff
@@ -2,61 +2,61 @@
 ## Schema of Built-in Functions
 | Class name | Function name or alias | Query example | Output schema |
```

Comment:
this is a lot of change... Shall we upper case everything except for string literals?

Reply:
Yeah, I agree it seems a lot of the diffs are related to this, as well as the whitespace after , and around -/+, etc...
I think our first goal is stability, our second reducing diffs to the existing algorithm.
Note that Oracle does not put in ANY whitespace. They simply squish everything together.

Reply:
> this is a lot of change... Shall we upper case everything except for string literals?

If you look at other places (not the diff w/ existing pretty printing), just output of the original SQL text looks more natural from my point of view.